### ISSUE TITLE
Magic Item with rarity Common breaks the GraphQL query

### ISSUE THREAD

**Issue by Rmulorm** [5e-bits/5e-srd-api#331]
When querying the API form Magic Items using GraphQL, if there's a Magic Item with Common Rarity the API returns an error message like this:
```json
{
  "errors": [
    {
      "message": "Enum \"MagicItemRarity\" cannot represent value: \"COMMON\"",
      "locations": [
        {
          "line": 6,
          "column": 5
        }
      ],
      "path": [
        "magicItems",
        41,
        "rarity"
      ],
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR",
        "exception": {
          "message": "Enum \"MagicItemRarity\" cannot represent value: \"COMMON\"",
          "stacktrace": [
            "GraphQLError: Enum \"MagicItemRarity\" cannot represent value: \"COMMON\"",
            "    at GraphQLEnumType.serialize (/app/node_modules/graphql/type/definition.js:1003:13)",
            "    at completeLeafValue (/app/node_modules/graphql/execution/execute.js:653:37)",
            "    at completeValue (/app/node_modules/graphql/execution/execute.js:578:12)",
            "    at completeValue (/app/node_modules/graphql/execution/execute.js:556:21)",
            "    at /app/node_modules/graphql/execution/execute.js:469:16",
            "    at processTicksAndRejections (node:internal/process/task_queues:96:5)",
            "    at async Promise.all (index 3)",
            "    at async Promise.all (index 41)"
          ]
        }
      }
    }
  ],
  "data": {
    "magicItems": null
  }
}
```

Steps to Reproduce:
Query the APi using the Apollo sandbox explorer with the following query with the following Variables:
Query:
```graphql
query MagicItemsQuery($equipmentCategory: StringFilter, $order: MagicItemOrder) {
  magicItems(equipment_category: $equipmentCategory, order: $order) {
    index
    name
    desc
    rarity
    equipment_category {
      index
      name
    }
  }
}
```

Variables
```json
{
  "equipmentCategory": null,
  "order": {
    "by": "EQUIPMENT_CATEGORY",
    "direction": "ASCENDING",
    "then_by": {
      "by": "NAME",
      "direction": "ASCENDING",
    }
  }
}
```

**Comment by techwraith** [comment]
I ran into this too! When can #332 get merged?

**Comment by techwraith** [comment]
Looks like this specific error was fixed when @bagelbits, @fergcb, and @abias1122 merged #332 it fixed this error, but now I'm getting this one: ```GraphQL error: Enum "MagicItemRarity" cannot represent value: "ARTIFACT"```

**Comment by techwraith** [comment]
Here's a PR to fix the latest error: #334

**Comment by bagelbits** [comment]
Okay this should be properly fixed.
